source/partials/drupal-8-convert-to-composer.md

```diff
@@ -24,7 +24,7 @@ To minimize issues, these steps make the codebase changes in a new branch:
   If you continue to encounter the error, use HTTPS to add the remote:
 
    ```bash{outputLines:2}
-   git remote add ic https://github.com/pantheon-upstreams/drupal-recommended.git && git fetch ic && git checkout --no-track -b composerify ic/master
+   git remote add ic https://github.com/pantheon-upstreams/drupal-composer-managed.git && git fetch ic && git checkout --no-track -b composerify ic/master
```

Comment:
Another `ic/main`

Reply:
Addressed in the previous commit [979fe19](https://github.com/pantheon-systems/documentation/pull/7292/commits/979fe19a942a7302db1f51cdee3cd8cd4a3e6683)